source/creatureeffect/CreatureEffectSpeedChange.cpp

```diff
@@ -59,6 +59,7 @@ void CreatureEffectSpeedChange::applyEffect(Creature& creature)
 
 void CreatureEffectSpeedChange::releaseEffect(Creature& creature)
 {
+    //FIXME: sanitizer reports that this is called at least once on something that is not a creature.
```

Comment:
I don't believe this is called with something else than a creature. I would guess that somehow, we call this in the creature destructor and the object is not considered a creature anymore